test/correctness/tracing.cpp

```diff
@@ -121,7 +120,7 @@ int main(int argc, char **argv) {
 
     // The golden trace, recorded when this test was written
     event correct_trace[] = {
-        {102, 8, 0, 32, 1, 0, 0, {0, 0, 0, 0}, {0.000000, 0.000000, 0.000000, 0.000000}},
```

Comment:
Maybe not for this PR, but it seems like these should use the actual named values from https://github.com/halide/Halide/blob/master/src/runtime/HalideRuntime.h#L279 so these don't get stale.